public/js/app/templates/bookmarkletTemplate.handlebars

```diff
@@ -101,6 +101,21 @@
   <div class="row">
     <div class="col-md-12">
       <h1>Share on FreeFeed</h1>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12">
+      <label for="bookmarklet-send-to-select">To:</label><br/>
+      <select id="bookmarklet-send-to-select" multiple="multiple" class="p-sendto-select">
+        <option selected="selected" value="{{controller.session.currentUser.username}}" class="p-sendto-selected">My feed</option>
+        {{#each feed in controller.session.currentUser.feeds}}
+          <option value="{{feed.user.username}}" class="p-sendto-option">{{feed.user.screenName}} ({{feed.user.username}})</option>
+        {{/each}}
+      </select>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12">
```

Comment:
Looks good, but since we are adding labels ("To"), now it looks off that the "title" is the only one without a label. We should either remove the label "To" and hope that it's clear from its initial value(s) what is it, or we should add "Title" and risk that form gets really bloated. (I'm inclined to the former option.)